jpegenc/subblocks/quantizer/__init__.py

```diff
@@ -0,0 +1,18 @@
+
+from __future__ import absolute_import
+
+from .quantiser_core import (QuantInputStream, QuantOutputStream,
+                             QuantConfig, quantizer,)
```

Comment:
In the other subblocks we don't use **top** for the top-level.  The internal can use _core.

You need to be consistent with spelling: quantizer vs. quantiser.  My spell checker likes quantizer and does not like quantiser :)  The root word is quantize.  I suspect it is a regional thing but either way we need to be consistent (one or the other).